### ISSUE TITLE
ModuleNotFoundError: No module named 'pip.req'  during installing pip

### ISSUE THREAD

**Issue by jxltom** [heroku/heroku-buildpack-python#700]
Following error occurs during installing pip

```
-----> Installing pip        
remote: Traceback (most recent call last):        
remote:   File "/tmp/buildpackKXfaB/vendor/pip-pop/pip-diff", line 15, in <module>        
remote:     from pip.req import parse_requirements        
remote: ModuleNotFoundError: No module named 'pip.req'        
remote: mount: can't find failure.bad-requirements in /etc/fstab or /etc/mtab 
```

This error occurs during every new rebulding.

**Comment by jxltom** [comment]
This should be related with pip 10 (https://github.com/heroku/heroku-buildpack-python/issues/680). 

And this should be fixed by https://github.com/heroku/heroku-buildpack-python/pull/683. I don't why it is still not working.

**Comment by CaseyFaist** [comment]
This should be resolved with the work you submitted being merged at #769 👍
